docs/man/docker-run.1.md

```diff
@@ -232,8 +232,9 @@ and foreground Docker containers.
 default is false. If the operator uses -P (or -p) then Docker will make the
 exposed port accessible on the host and the ports will be available to any
 client that can reach the host. When using -P, Docker will bind the exposed
-ports to a random port on the host between 49153 and 65535. To find the
-mapping between the host ports and the exposed ports, use **docker port**.
+port to a random port on the host within an *ephemeral port range* defined by
```

Comment:
how about `Docker will bind any exposed port to a random ..` as there may be more than one.